libmachine/drivers/plugin/localbinary/utils_linux.go

```diff
@@ -0,0 +1,32 @@
+package localbinary
```

Comment:
This file also needs build tags 

```
//go:build !windows
// +build !windows
```

Reply:
Fixed.  

I also removed the `+build` tag which is the older way of doing conditional compilation before GO 1.17